CMake/External_OpenCV.cmake

```diff
@@ -200,6 +236,11 @@ if (CMAKE_COMPILER_IS_GNUCC AND NOT CMAKE_CXX_COMPILER_VERSION VERSION_LESS 6)
   list(APPEND OpenCV_EXTRA_BUILD_FLAGS -DENABLE_PRECOMPILED_HEADERS:BOOL=OFF)
 endif()
 
+# OpenCV 3.3.0 has an option to enable C++ 11
+if (fletch_BUILD_CXX11)
+  list(APPEND OpenCV_EXTRA_BUILD_FLAGS -DENABLE_CXX11:BOOL=ON)
```

Comment:
I just confirmed that OpenCV 3.3.0 does can build without C++11. 

The issue is if you enable C++11 in fletch but dont set OpenCV's `ENABLE_CXX11` flag, you will get build errors. I'm working on confirming what these are.